### ISSUE TITLE
False Positive on [amqp-client] [activemq-client]

### ISSUE THREAD

**Issue by alero** [dependency-check/DependencyCheck#2334]
- The location of the dependency (Maven GAV, URL to download the dependency, etc.)
https://mvnrepository.com/artifact/com.rabbitmq/amqp-client/5.5.3
https://ossindex.sonatype.org/component/pkg:maven/org.apache.activemq/activemq-client@5.15.10

All of these are server issues reported on a client library, is there not a better way of filtering it by default?

False positive on library amqp-client.jar - reported as `cpe:2.3:a:pivotal_software:rabbitmq:5.5.3`
```xml
<dependency>
   <groupId>com.rabbitmq</groupId>
   <artifactId>amqp-client</artifactId>
   <version>5.5.3</version>
</dependency>
```
AND

False positive on library a:apache:activemq-client.jar - reported as `cpe:2.3:a:apache:activemq:5.15.10`
```xml
<dependency>
   <groupId>org.apache.activemq</groupId>
   <artifactId>activemq-client</artifactId>
   <version>5.15.10</version>
</dependency>
```

**Comment by jeremylong** [comment]
Unfortunately - we do not have a good way to disambiguate client libraries from servers - and worse in some rare cases the same CPE is used for both. the client and server

**Comment by jeremylong** [comment]
The suppression rule for this has been removed as it was deemed to generic. See https://github.com/jeremylong/DependencyCheck/issues/2483
